src/libraries/System.Private.CoreLib/src/System/Number.Formatting.cs

```diff
@@ -584,6 +586,13 @@ private static int GetFloatingPointMaxDigitsAndPrecision(char fmt, ref int preci
             return null;
         }
 
+        public static string FormatHalf(Half value, string? format, NumberFormatInfo info)
+        {
+            Span<char> stackBuffer = stackalloc char[CharStackBufferSize];
+            var sb = new ValueStringBuilder(stackBuffer);
```

Comment:
Nit: this can now be:
```suggestion
            var sb = new ValueStringBuilder(stackalloc char[CharStackBufferSize]);
```

Reply:
(although I see it's just copying the same formatting as used elsewhere in the file, so ok :))

Reply:
Cool. Leaving this as is then.

Reply:
Mind still changing it? I've fixed all the others here https://github.com/dotnet/runtime/pull/37795 :-)

Reply:
Done :).